doc/source/whats_new.rst

```diff
@@ -918,19 +920,19 @@ Version 0.2
 Changelog
 ~~~~~~~~~
 
-   - New stats functions for FDR correction and Bonferroni by `Alex Gramfort`_.
+    - New stats functions for FDR correction and Bonferroni by `Alex Gramfort`_.
 
-   - Faster time-frequency using downsampling trick by `Alex Gramfort`_.
+    - Faster time-frequency using downsampling trick by `Alex Gramfort`_.
 
-   - Support for volume source spaces by `Alex Gramfort`_ (requires next MNE release or nightly).
+    - Support for volume source spaces by `Alex Gramfort`_ (requires next MNE release or nightly).
 
-   - Improved Epochs handling by `Martin Luessi`_ (slicing, drop_bad_epochs).
+    - Improved Epochs handling by `Martin Luessi`_ (slicing, drop_bad_epochs).
 
-   - Bug fix in Epochs + ECG detection by Manfred Kitzbichler.
+    - Bug fix in Epochs + ECG detection by Manfred Kitzbichler.
 
-   - New pick_types_evoked function by `Alex Gramfort`_.
+    - New pick_types_evoked function by `Alex Gramfort`_.
 
-   - SourceEstimate now supports algebra by `Alex Gramfort`_.
+    - SourceEstimate now supports algebra by `Alex Gramfort`_.
```

Comment:
I just squashed the spacing for this pr. I was thinking having it as a separate pr. i don't think git blame is necessary with the whats_new IMO.